core/events/tests/test_classic_integration.py

```diff
@@ -63,19 +87,19 @@ def test_classic_workflow(self):
             # /start: Submitter completes the start submission page.
             license_uri = 'http://creativecommons.org/publicdomain/zero/1.0/'
             submission, stack = events.save(
-                events.VerifyContactInformation(creator=self.submitter),
+                events.VerifyContactInformation(creator=submitter),
                 events.AssertAuthorship(
-                    creator=self.submitter,
+                    creator=submitter,
                     submitter_is_author=True
                 ),
                 events.SelectLicense(
-                    creator=self.submitter,
+                    creator=submitter,
                     license_uri=license_uri,
                     license_name='CC0 1.0'
                 ),
-                events.AcceptPolicy(creator=self.submitter),
+                events.AcceptPolicy(creator=submitter),
                 events.SetPrimaryClassification(
-                    creator=self.submitter,
+                    creator=submitter,
                     category='cs.DL'
```

Comment:
We discussed this: in ARXIVNG-458 it would be good to parameterize all of the data.